config-linux.md

```diff
@@ -531,35 +528,26 @@ The following parameters can be specified to set up seccomp:
     * `SCMP_ARCH_S390X`
     * `SCMP_ARCH_PARISC`
     * `SCMP_ARCH_PARISC64`
-
 * **`syscalls`** *(array of objects, OPTIONAL)* - match a syscall in seccomp.
-
     While this property is OPTIONAL, some values of `defaultAction` are not useful without `syscalls` entries.
     For example, if `defaultAction` is `SCMP_ACT_KILL` and `syscalls` is empty or unset, the kernel will kill the container process on its first syscall.
-
     Each entry has the following structure:
-
     * **`names`** *(array of strings, REQUIRED)* - the names of the syscalls.
         `names` MUST contain at least one entry.
     * **`action`** *(string, REQUIRED)* - the action for seccomp rules.
         A valid list of constants as of libseccomp v2.3.2 is shown below.
-
```

Comment:
I think we want to keep this pre-list (and the later post-list) blank lines too.  There are a number of other instances of this as well.